db/migrate/20150710110707_add_missing_fields_and_indexes_to_cloud_subnets.rb

```diff
@@ -0,0 +1,13 @@
+class AddMissingFieldsAndIndexesToCloudSubnets < ActiveRecord::Migration
+  def change
+    add_column :cloud_subnets, :cloud_tenant_id,                :bigint
+    add_column :cloud_subnets, :dns_nameservers,                :string
+    add_column :cloud_subnets, :ipv6_router_advertisement_mode, :string
+    add_column :cloud_subnets, :ipv6_address_mode,              :string
+    add_column :cloud_subnets, :extra_attributes,               :text
```

Comment:
@ladas, should you add `allocation_pools` here?  Looking at some output from OSP7 Neutron:

```
irb(main):015:0> neutron.subnets
=>   <Fog::Network::OpenStack::Subnets
    filters={}
    [
                  <Fog::Network::OpenStack::Subnet
        id="aaabbb-guid",
        name="public_sub",
        network_id="aaabbb-guid",
        cidr="10.1.1.0/23",
        ip_version=4,
        gateway_ip="10.1.1.254",
        allocation_pools=[{"start"=>"10.1.1.150", "end"=>"10.1.1.170"}],
        dns_nameservers=[],
        host_routes=[],
        enable_dhcp=false,
        tenant_id="fff555-guid"
      >    
    ]
  >
```

As long as we're collecting more information from Neutron, seems like it would make more sense to pick up allocation pools that might be useful during provisioning.

/cc @gmcculloug